ops/fhir_validations/validations.py

```diff
@@ -82,7 +83,11 @@ def validate_resources(validator_path: str, version: str, ignore_list: dict, fil
     output_lines = output.split('\n')
 
     file_name = "loading_output"
-    errors_per_file = {}
+
+    if len(files) == 1:
+        file_name = "/" + os.path.basename(files[0])
+
+    errors_per_file = {file_name: []}
```

Comment:
when we have multiple files, won't we always have a "loading_output" entry in the map?

Reply:
I think an empty list would get removed in `filter_errors` since it has no errors to add to the filtered dictionary.  So it would be harmless.

Reply:
Correct, given that no errors will be saved to that key, the later code just skips over it:
https://github.com/CMSgov/beneficiary-fhir-data/blob/master/ops/fhir_validations/validations.py#L52-L55
specifically on line 55, there will be no entries for that key